README.adoc

```diff
@@ -61,6 +61,33 @@ If you want to connect devices on external networks to HiveMQ CE, please make su
 
 Just in time builds for current branches on this repository and for specific commits are available https://hivemq.github.io/nightly-builds/[here].
 
+== Run with Docker
+
+The current state of the `master` branch will be pushed to the https://hub.docker.com/r/hivemq/hivemq-ce[hivemq/hivemq-ce] repository.
+To execute this image, simply run the following command:
+
+[source,bash]
+----
+docker run --name hivemq-ce -e HIVEMQ_LOG_LEVEL=INFO -d -p 1883:1883 /hivemq/hivemq-ce
```

Comment:
The "-e HIVEMQ_LOG_LEVEL=INFO" is the default anyway. So i think we can skip it in the default command.

Reply:
i would've left it there simply to demonstrate how to manipulate the log level.

Reply:
@schaebo what do you think?

Reply:
Let's try to keep the command as simple as possible.

Maybe add a line in the "Build Docker image manually" like.
To change the default log level you can set the environment variable `HIVEMQ_LOG_LEVEL` on the container.